### PR TITLE
Feedback idr fixes

### PR DESCRIPTION
**Pull request by will-moore**
Various fixes from feedback on the IDR gallery.

To test:
 - Mapr search results should show study Title under the Title column of the table (not description)
 - Hovering over a Study on the home page or search page (filtering by Study Attributes) should highlight the text in a lighter (higher contrast) blue
 - Page footer should be full width of the page (same as on idr.openmicroscopy.org)
 - Header links: About is moved to the right and contains all other links except submission. See https://github.com/IDR/idr.openmicroscopy.org/pull/55#issuecomment-499447719
 - Longer text for subheading. Does it read OK?
 - "Publication Author" auto-complete shouldn't include first names and should get correct Surname (some examples to try in comment below, and some known issues)
 - Placeholder text in auto-complete updated to indicate that you need to type to filter (for Study Attributes) and type to get any results for Image Attributes.

### CODE REVIEW

**Comment by will-moore** [comment]
Trying to improve regex to get surname only from all Publication Authors...
Mix of names (produced by splitting the ```Publication Authors``` value by ```,``` or ``` and ``` or ``` & ```). E.g:
 - Francesco Paolo Casale
 - Carpenter AE.
 - Horvath P
 - Fiona M. Watt
 - Jean-Karim Hériché
 - Øyvind Ødegård-Fougner
 - Li K
 - de Marco A
 - Arias-Garcia M
 - M. Julius Hossain
 - Petri Seiler K

I'm not even sure what the surname is for the last 2.
If anyone can figure out a rule that gives the correct surname from each of these cases, please let me know. cc @sbesson @francesw.
Currently I'm splitting on ```' '```, filtering for words that contain a lowercase. Then if there are 1 words, that's the surname, if there are 2 or more words, I ignore the first word (first name) and join the rest.
e.g. Francesco Paolo Casale -> Paolo Casale
But also Petri Seiler K -> Seiler and M. Julius Hossain -> Hossain.

**Comment by manics** [comment]
If you've figured out a rule we could use it to automatically extract the surnames and enter them as individual key-value pairs.

**Comment by sbesson** [comment]
Re publication authors, my inclination would be to work towards unifying the formatting of the `Publication Authors` key in the study files and their representation in IDR. @manics has proposed a few ways to store the data as map annotations. For the study files, I can certainly conceive using the Pubmed style everywhere especially since we use PubMed ID as the primary publication identifier i.e. `Last Name 1 <Initials 1>, Last Name 2 <Initials 2>`. Probably something worth discussing on @francesw return.

**Comment by joshmoore** [comment]
* :+1: redirect
 *  :+1: "Mapr search results should show study Title"
 * :+1: Hovering over a Study on the home page or search page 
 * :+1: Page footer should be full width of the page 
 * :+1: Header links: About is moved to the right...
    - not that the top level and "overview" go to the same page. same for submission. We (You?) got rid of this on www.o.org
 * "Longer text for subheading. Does it read OK?" You mean "The IDR is a public repository...". Reads ok as it is, but I still think we should change this to what the user wants to DO now that they are here.
 * :+1: "Publication Author" auto-complete shouldn't include first names - looks last name-ish.
    - but I noticed that the very long list displays a bit oddly over the footer. Don't know if that can be made scrollable?
 * :+1: Placeholder text in auto-complete updated to indicate ...

**Comment by joshmoore** [comment]
tl;dr - happy for this to be merged if there aren't any easy fixes that @will-moore wants to get in.

**Comment by will-moore** [comment]
Nothing else to add right now, thanks.
